pyproject.toml

```diff
@@ -11,7 +11,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "~3.8"
+python = "~3.9"
```

Comment:
I think this will break projects written in python 3.8. Better to leave it unchanged

Reply:
ok